### PR TITLE
Update Alerting Docs

### PR DESCRIPTION
**Pull request by rekt0x**
Remove the outdated Telegram monitoring bot and replace it with Vordr, as discussed in https://gitlab.com/thorchain/devops/node-launcher/-/merge_requests/691.

### CODE REVIEW

File: thornodes/alerting.md [hunk 1]
```diff
@@ -1,255 +1,47 @@
...
-Next you can specify the IP of the Thornode that you want to watch in the `THORCHAIN_NODE_IP` environment variable.
-
-Set it to `localhost` to listen a node on your own machine.
-
-Leave this environment variable empty or don't even set it to use IPs of the testnet seed nodes from [https://testnet-seed.thorchain.info](https://testnet-seed.thorchain.info/).
+> A Vǫrðr or warden is a guardian spirit who will follow the soul of a living person from birth until death.
 
 {% hint style="info" %}
-_Please note, if you leave `THORCHAIN_NODE_IP` empty, IP specific monitoring won't take effect \(no check for increasing block height, midgard API and catch up status\). Because we rotate through available testnet seed IPs, we would compare data of different nodes and send incorrect alerts._
+Vǫrðr is a community developed monitoring app for THORNodes. It's fully open-source at [https://github.com/sourcapital/vordr](https://github.com/sourcapital/vordr).
```
Comment (Orion-9R) [inline thornodes/alerting.md:24]: My 2 sats - this is fine as long as it is very clear that this is 3rd party software outside of the scope of THORSec.

Comment (rekt0x) [reply thornodes/alerting.md]: I added an extra disclamer that said that.

**Comment by Orion-9R** [comment]
LGTM - I don't have approval access.

@GrassRootsCrypto - can you take a look and approve + merge if good to go?

**Review by GrassRootsCrypto — APPROVED** [review]
Looks good @Orion-9R

**Comment by GrassRootsCrypto** [comment]
@Orion-9R - I don't seem to have rights to merge.

**Pull request merged**